lib/css/exports/_stacks-mixins.less

```
@@ -29,6 +29,27 @@
             appearance: @all;
 }
 
+//  ===========================================================================
+//  --  FOCUS VISIBLE
+//      Powered by https://github.com/WICG/focus-visible while we wait for better
+//      :focus-visible support
+//  ---------------------------------------------------------------------------
+.focus(@rules) {
+    // This will hide the focus indicator if the element receives focus via the mouse
+    .js-focus-visible & {
+        &:focus:not(.focus-visible) {
+            outline: none;
+        }
+    }
+
+    // Adds a strong focus indicator for keyboard focus.
+    .js-focus-visible & {
+        &.focus-visible {
+            outline: none;
+            @rules();
+        }
+    }
+}
```

```suggestion

    &:focus-visible {
        outline: none;
        @rules();
    }
}
```

Since Chrome supports this and Firefox is on the way, should we include actual support for the selector?

---

+1. [caniuse](https://caniuse.com/css-focus-visible) reports that Safari is the last holdout (Edge legacy too, I guess). Maybe we can figure out a progressive enhancement scenario for those two?